### PR TITLE
Fix major bugs and create interactive installation script for new users

### PR DESCRIPTION
**Pull request by chc5**

### CODE REVIEW

**Comment by chc5** [comment]
@yliu0715 run this on gcp and tell me if it's fine or not

**Comment by yliu0715** [comment]
error

**Comment by yliu0715** [comment]
list

Everything except the cv image built correctly.

**Comment by chc5** [comment]
check again @yliu0715

**Comment by chc5** [comment]
@yliu0715 again

File: src/kafka/notifications/src/index.ts [hunk 1]
```diff
@@ -3,26 +3,27 @@ import * as fs from 'fs';
...
+    autoCommit: false,
+  });
 
 const getBlacklisted = 'SELECT * FROM blacklist WHERE name = ?';
 const getUserID = 'SELECT user_id FROM camera_users WHERE camera_id = ?';
 const getEmail = 'SELECT email FROM users_id WHERE user_id = ?';
 const getImg = 'SELECT frame FROM cv_frames WHERE frame_id = ?';
 const storeNotif =
-    'INSERT INTO notif (date, type, email, name, frame_id) VALUES (%s, %s, %s, %s, %s)';
+  'INSERT INTO notif (date, type, email, name, frame_id) VALUES (%s, %s, %s, %s, %s)';
```
Comment (yliu0715) [inline src/kafka/notifications/src/index.ts:24]: Change the `%s` to `?`

**Pull request merged**